### PR TITLE
Feature/#3 fifunmitimothy1 user registration

### PR DESCRIPTION
**Pull request by fifunmitimothy1**

### CODE REVIEW

**Review by asif-kamal — DISMISSED** [review]

**Review by anji-annavarapu — DISMISSED** [review]

File: src/main/java/com/rev_connect_api/controllers/HomeController.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class HomeController {
 
     @PostMapping
     public ResponseEntity<String> registerUser(@RequestBody User user) {
-        System.out.println("*************************Startig point*****************");
+        System.out.println("*************************Starting point*****************");
```
Comment (themellowpea) [inline src/main/java/com/rev_connect_api/controllers/HomeController.java:21]: You can remove this

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-server.port=8080
+server.port=5000
```
Comment (themellowpea) [inline src/main/resources/application.properties]: change to `8080`
```suggestion
server.port=8080
```

File: pom.xml [hunk 3]
```diff
@@ -70,6 +70,7 @@
      		<artifactId>jaxb-api</artifactId>
      		<version>2.4.0-b180830.0359</version>
    		</dependency>
+
```
Comment (themellowpea) [inline pom.xml]: will probably scrub everything for formatting soon

```suggestion
```

**Review by themellowpea — CHANGES_REQUESTED** [review]
We can use this for CORS.

File: src/main/java/com/rev_connect_api/controllers/HomeController.java (hunk 1 above)
Comment (anji-annavarapu) [reply src/main/java/com/rev_connect_api/controllers/HomeController.java:21]: i just wanna keep this for testing, in my next commit i will do this

**Review by themellowpea — APPROVED** [review]

**Review by asif-kamal — APPROVED** [review]

**Review by themellowpea — CHANGES_REQUESTED** [review]
Fix merge conflicts before we can review.

**Pull request closed without merging**